### PR TITLE
Consolidate lantern.yaml script generation

### PR DESCRIPTION
**Pull request by myleshorton**

### CODE REVIEW

File: src/github.com/getlantern/flashlight/client/reverseproxy.go [hunk 1]
```diff
@@ -45,7 +45,8 @@ func (client *Client) newReverseProxy() (*httputil.ReverseProxy, error) {
 	// set that in the Transport RoundTrip call above.
 	dialer, conn, err := client.getBalancer().TrustedDialerAndConn()
 	if err != nil {
-		log.Errorf("Could not get balanced dialer", err)
+		// The internal code has already reported an error here.
+		log.Debugf("Could not get balanced dialer", err)
```
Comment (fffw) [inline src/github.com/getlantern/flashlight/client/reverseproxy.go]: No formatter supplied

File: src/github.com/getlantern/tarfs/tarfs_test.go [hunk 2]
```diff
@@ -10,6 +10,34 @@ import (
...
+		t.Fatalf("Unable to encode to tar string: %v", err)
+	}
+
+	fs, err := New(tarStringToBytes(t, tarString), "localresources")
+	if err != nil {
+		t.Fatalf("Unable to open filesystem: %v", err)
+	}
+
+	a, err := fs.Get("empty.txt")
+	if assert.NoError(t, err, "empty.txt should have loaded") {
```
Comment (fffw) [inline src/github.com/getlantern/tarfs/tarfs_test.go]: `localresources/empty.txt` is not added to repo.

File: src/github.com/getlantern/tarfs/tarfs.go [hunk 3]
```diff
@@ -111,6 +111,12 @@ func (fs *FileSystem) SubDir(dir string) *FileSystem {
 	}
 }
 
+// GetIgnoreEmpty is the same as Get, but it ignores local file system files
+// if they're empty.
+func (fs *FileSystem) GetIgnoreEmpty(p string) ([]byte, error) {
```
Comment (fffw) [inline src/github.com/getlantern/tarfs/tarfs.go]: The function name is a bit misleading. Can we make the name more explicit, say, `GetIgnoreLocalEmpty`, or tweak `get` a bit to also apply `ignoreempty` flag to files in-memory?

File: src/github.com/getlantern/tarfs/tarfs.go [hunk 4]
```diff
@@ -119,6 +125,17 @@ func (fs *FileSystem) SubDir(dir string) *FileSystem {
 // Note - the implementation of local reads is not optimized and is primarily
 // intended for development-time usage.
 func (fs *FileSystem) Get(p string) ([]byte, error) {
+	return fs.get(p, false)
+}
+
+// Get returns the bytes for the resource at the given path. If this FileSystem
```
Comment (fffw) [inline src/github.com/getlantern/tarfs/tarfs.go]: obsolete comment here

**Comment by myleshorton** [comment]
Going to create a new one for release-2.0.1

**Pull request closed without merging**